README.md

```diff
@@ -118,6 +157,15 @@ You can check the [Docker](https://docs.docker.com/reference/builder/) documenta
 
 Once the container is ready you can get in touch with us so we can make the appropriate arrangements to make your container available to everyone in the community by giving an automated build system.
 
+### 3.3. How to create a rkt based Biocontainer?
```

Comment:
Do we want to advertise this? quay.io supports both containers as soon as you deploy your container on there service. You get the conversion for Docker -> rkt for free (with some limitations).